apps/web/templates/email/email.html

```diff
@@ -1,198 +1,192 @@
-             </tr> 
-           </table> </td> 
-         </tr> 
-       </table>  
-       <table class="es-content" cellspacing="0" cellpadding="0" align="center" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;table-layout:fixed !important;width:100%;"> 
-         <tr style="border-collapse:collapse;"> 
-          <td style="padding:0;Margin:0;background-color:#F6F6F6;" bgcolor="#f6f6f6" align="center"> 
-           <table class="es-content-body" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;background-color:transparent;" width="640" cellspacing="0" cellpadding="0" align="center"> 
-             <tr style="border-collapse:collapse;"> 
-              <td align="left" style="Margin:0;padding-left:20px;padding-right:20px;padding-top:30px;padding-bottom:30px;"> 
-               <table width="100%" cellspacing="0" cellpadding="0" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;"> 
-                 <tr style="border-collapse:collapse;"> 
-                  <td width="600" valign="top" align="center" style="padding:0;Margin:0;"> 
-                   <table width="100%" cellspacing="0" cellpadding="0" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;"> 
-                     <tr style="border-collapse:collapse;"> 
-                      <td style="padding:0;Margin:0;display:none;" align="center"></td> 
-                     </tr> 
-                   </table> </td> 
-                 </tr> 
-               </table> </td> 
-             </tr> 
-           </table> </td> 
-         </tr> 
-       </table> </td> 
-     </tr> 
-   </table> 
-  </div>  
- </body>
-</html>
\ No newline at end of file
+  </head>
+  <body class="" style="background-color: #f6f6f6; font-family: 'open sans', 'helvetica neue', helvetica, arial, sans-serif; -webkit-font-smoothing: antialiased; font-size: 14px; line-height: 1.4; margin: 0; padding: 0; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;">
+    <table border="0" cellpadding="0" cellspacing="0" class="body" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; background-color: #f6f6f6;">
+      <tr>
+        <td style="font-size: 14px; vertical-align: top;">&nbsp;</td>
+        <td class="container" style="font-size: 14px; vertical-align: top; display: block; Margin: 0 auto; max-width: 580px; padding: 10px; width: 580px;">
+          <div class="content" style="box-sizing: border-box; display: block; Margin: 0 auto; max-width: 580px; padding: 10px;">
+
+            <!-- START CENTERED WHITE CONTAINER -->
+            <span class="preheader" style="color: transparent; display: none; height: 0; max-height: 0; max-width: 0; opacity: 0; overflow: hidden; mso-hide: all; visibility: hidden; width: 0;">This is preheader text. Some clients will show this text as a preview.</span>
```

Comment:
https://www.campaignmonitor.com/blog/email-marketing/2019/02/a-practical-guide-to-email-preheaders/

Instead of the placeholder text, write "Unread bookmarks from Bookie!"